frame/benchmarking/src/baseline.rs

```diff
@@ -75,6 +89,26 @@ benchmarks! {
 		assert!(hash != T::Hash::default());
 	}
 
+	sr25519_verification {
+		let i in 1 .. 100;
+
+		let public = SignerId::generate_pair(None);
+
+		let sigs_count: u8 = i.try_into().unwrap();
+		let msg_and_sigs: Vec<_> = (0..sigs_count).map(|j| {
+			let msg = vec![j, j];
+			(msg.clone(), public.sign(&msg).unwrap())
+		})
+		.collect();
+	}: {
+		msg_and_sigs.iter().for_each(|(msg, sig)| {
+			assert_eq!(
+				sig.verify(&msg[..], &public),
+				true
```

Comment:
would be good to verify that this is exactly what we want to measure versus a batch signature verification?

Reply:
I am most interested in the speed of the single verification like this https://github.com/paritytech/polkadot/blob/5afceea8b77d27ac6c3eaf500160e9f34807c26b/runtime/parachains/src/disputes.rs#L1221

Reply:
sounds good then